namespaces/live-1.cloud-platform.service.justice.gov.uk/uat-claim-criminal-injuries-compensation/resources/ecr.tf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cica_ecr_credentials" {
+  source     = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.1"
```

Comment:
The latest version of the ECR credentials module is 3.2 - please use that as the 'ref' value here.